release notes/v0.52.0.md

```diff
@@ -0,0 +1,141 @@
+k6 `v0.52.0` is here 🎉! Some special mentions included in this release:
+
+- [We switched from `goja` to our own fork named `sobek`](#switch-goja-to-our-own-fork-named-sobek-3775)
+- [Panics are no longer being captured](#panics-are-no-longer-being-captured-3777)
+- [The `web-dashboard` built-in output becomes `dashboard`](#the-web-dashboard-built-in-output-becomes-dashboard-xk6-dashboard163)
+- [We've added experimental support for TypeScript and ES6+](#experimental-support-for-typescript-and-es6-using-esbuild-3738)
+- [`k6/browser` has graduated from an experimental module](#k6browser-has-graduated-from-an-experimental-module-3793)
+- [`k6/browser` has now a fully Async API](#k6browser-has-now-a-fully-async-api-browser428)
+
+## Breaking changes
+
+### Switch `goja` to our own fork named `sobek` [#3775](https://github.com/grafana/k6/pull/3775)
+
+To accelerate the development speed and bring [ECMAScript Modules (ESM)] support to k6 earlier (https://github.com/grafana/k6/issues/3265),
+we have decided to create a fork of the [`goja`](https://github.com/dop251/goja/) project under the Grafana GitHub organization,
+named [`sobek`](https://github.com/grafana/sobek).
+
+So, since this release, k6 (and its extensions) starts using `sobek` instead of the original `goja`, for all (of the
+publicly exposed parts of the API) except for a couple of packages that are only used internally by k6.
+
+Find further details in [#3772](https://github.com/grafana/k6/issues/3772) and [#3773](https://github.com/grafana/k6/issues/3773).
+
+### Panics are no longer being captured [#3777](https://github.com/grafana/k6/pull/3777)
+
+Since this release, Go panics are no longer being captured by k6. This means that if a panic occurs while running a test,
+the k6 process will crash, and the panic stack trace will be printed to the console.
+
+We decided to change this behavior because it's something that was left from the past, as a safeguard, but it's not as
+good as it might seem, as for most cases with multiple goroutines/async it is not enough, and also makes a bunch of potential
+bugs seem like less of an issue.
```

Comment:
```suggestion
We decided to change this behavior because it's something that was left from the past as a safeguard, but it's not as
good as it might seem. For most cases with multiple goroutines/async, it's not enough and also makes a bunch of potential
bugs seem like less of an issue.
```